amida-tech/cql-execution
# Create actuator endpoint

Uses ExpressJS and the Node module express-actuator to create a simple actuator endpoint.

# Related Tickets

- [SAR-148](https://jira.amida-tech.com/browse/SAR-148)

# How to Test

`npm install` (To setup new dependencies)
`yarn start`

Check in browser
`http://localhost:5001/health`
`http://localhost:5001/info`
`http://localhost:5001/metrics`
